### PR TITLE
Added KyLeggiero.me, Blog.KyLeggiero.me, and Resume.BenLeggiero.me

### PR DESCRIPTION
**Pull request by KyLeggiero**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -96,6 +96,7 @@ blog.aractus.com
 blog.counter-strike.net
 blog.dota2.com
 blog.ja-ke.tech
+blog.kyleggiero.me
```
Comment (Gusted) [inline src/config/dark-sites.config:99]: You don't have to add any subdomain. the domain itself is sufficient.

**Review by Gusted — CHANGES_REQUESTED** [review]
That also should resolve the issue with why the tests are failing.

File: src/config/dark-sites.config (hunk 1 above)
Comment (KyLeggiero) [reply src/config/dark-sites.config:99]: Oh interesting! I was just following the lead of others in the file. Will push that, thank you

Comment (KyLeggiero) [reply src/config/dark-sites.config:99]: Oh interesting! I was just following the lead of others in the file. Will push that, thank you

**Review by Gusted — APPROVED** [review]

File: src/config/dark-sites.config (hunk 1 above)
Comment (KyLeggiero) [reply src/config/dark-sites.config]: @Gusted it seems that blog.kyleggiero.me is now being treated as if it's not on this list. What happened?

**Pull request merged**